src/main/java/by/onliner/page/MainPage.java

```diff
@@ -0,0 +1,40 @@
+package by.onliner.page;
+
+import by.onliner.driver.DriverManager;
+import by.onliner.dto.Menu;
+import by.onliner.util.Waiter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+
+import static by.onliner.constants.OnlinerConstants.*;
+
+
+public class MainPage extends AbstractPage {
+
+    @FindBy (xpath = "//div[@data-index= '1']")
+    private WebElement catalogFirstItem;
```

Comment:
What is 'firstItem'? please use proper naming in a week you will forget what this name means

Reply:
Renamed to firtItemFromCatalogList

Reply:
Also bad naming, what is the first item? Try to get rid of it, it's not informative

Reply:
Either specify the item you want to add and pass the item name from the test (preferred) or retrieve a list of all catalog elements and choose any of them.
Anyway, if in a test you will need to add to the shopping cart a specific item you will add a separate element for it? but locators are almost the same